helpers

```diff
@@ -55,10 +55,10 @@ update_repo() {
 	fi
 	set +e
 	# Ensure that CLRVER exists and is a tag
-	git rev-parse --verify "${CLRVER}^{tag}" &> /dev/null
+	git rev-parse --verify "refs/tags/${CLRVER}^{tag}" &> /dev/null
 	if [ $? -eq 0 ]; then
 		# Always reset the _mix branch to be based off the tag
-		git checkout -B "${CLRVER}_mix" "${CLRVER}"
+		git checkout -B "${CLRVER}_mix" "refs/tags/${CLRVER}"
```

Comment:
Might want to clean after this too in order to avoid new bundle files not in git. Also this can fail if the branch already exists, should the return from the checkout command be tested?